sdk/communication/AzureCommunicationCommon/Tests/CommunicationTokenCredentialTests.swift

```diff
@@ -79,186 +79,8 @@ class CommunicationTokenCredentialTests: XCTestCase {
         }
     }
 
-    func test_DecodesToken() throws {
```

Comment:
Nope they are moved to other test files([AutoRefreshOnDemandTokenCredentialTests.swift](https://github.com/Azure/azure-sdk-for-ios/pull/1422/files#diff-539e412461fa6262b31bd4d6c874172604fe6d6acd3c4f6bee1afd4480fbf7cc), [AutoRefreshProactiveTokenCredentialTests.swift](https://github.com/Azure/azure-sdk-for-ios/pull/1422/files#diff-385fe5ea30cabf8d05762f27080e4e8c40967682febe0e90e521c76061a82a6a), [StaticTokenCredentialTests.swift](https://github.com/Azure/azure-sdk-for-ios/pull/1422/files#diff-dd6097e1a92f6916c37ebd59602b1a7f5d8af89084815decbea8bc8a0a461b95)), otherwise to have all the tests in the one file was failing for the following linting rule:
`Type body should span 250 lines or less excluding comments and whitespace`

Reply:
Got it. You could also just use a swiftlint disable statement :)